File: acmicpc/cpp/2525_오븐 시계.cpp
#include <stdio.h>

int main(void) {

    int hour, minute;
    int need_time;

    /** input */
    scanf("%d %d", &hour, &minute);
    scanf("%d", &need_time);

    minute += need_time;

    while(minute >= 60) {
        minute -= 60;
        hour ++;
    }
    hour %= 24;

    /** output */
    printf("%d %d\n", hour, minute);

    return 0;
}

File: acmicpc/python/2525_오븐 시계.py
hour, minute = input().split()
remained_minute = input()

hour = int(hour)
minute = int(minute)
remained_minute = int(remained_minute)

calc_hour = int(remained_minute / 60)
calc_minute = remained_minute % 60

minute += calc_minute
if minute >= 60:
    hour += 1
    minute %= 60
hour += calc_hour
if hour >= 24:
    hour %= 24

print(hour, minute)